plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/status-popover.tsx

```diff
@@ -7,6 +7,7 @@ import { useState } from '@wordpress/element';
 
 export default function StatusPopover( props: {
 	text: string;
+	level: 'error' | 'warning';
```

Comment:
I see this in a few places. Should we use an enum?

Reply:
Yes, makes sense. Implemented in https://github.com/woocommerce/woocommerce/pull/41066/commits/45d85c273054479a9fa3c67c7fc38949d8516146.